machine_learning_hep/steer_analysis.py

```diff
@@ -192,11 +190,16 @@ def do_entire_analysis(data_config: dict, data_param: dict, data_param_overwrite
         logger.info('existing directories must be deleted')
         for d in exdirs:
             print(f'rm -rf {d}')
+        delete = False
         if args.delete:
             ok = input('Do you want to delete these directories now (y/n)? ')
-            if ok.lower() == 'y':
-                while len(exdirs) > 0:
-                    shutil.rmtree(exdirs.pop())
+            delete = ok.lower() == 'y'
+        if args.delete_force:
+            delete = True
+        if delete:
+            logger.warning("Deleting output directories")
```

Comment:
I don't think this should be a warning.

Reply:
Which level do you prefer? `info` or `debug`?

Reply:
Either is fine, I don't need to see the message but I can live with it, I just don't consider it a reason to warn about anything.